### DISCUSSION TITLE
Custom Forms: get_invoice endpoint custom form field data

### DISCUSSION THREAD

**Discussion by jayparalejas** [btcpayserver/btcpayserver#5483] [Technical Support] [answered]
Is there any way to fetch the custom form field data from the official api?

https://docs.btcpayserver.org/Forms/

Searched everywhere but can't seem to figure out how the custom form field is exposed in the endpoint.

**Comment by dstrukt** [comment]
Ping @dennisreimann

**Reply by dennisreimann** [reply, marked as answer]
Answered here, please no crossposts. #5484
